.github/workflows/main.yml

```diff
@@ -4,40 +4,34 @@ on:
   push:
     branches:
     - master
+    - main # ✊🏿
+
+env:
+  CI: true
+  SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+  HUSKY_SKIP_INSTALL: true
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [12.x]
     steps:
-      - uses: actions/checkout@v1
-
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v1
-        with:
-          node-version: ${{ matrix.node-version }}
+    - uses: actions/checkout@v2
 
-      - name: Check syntax
-        run: |
-          yarn install
-          yarn lint
+    - name: Check syntax
+      run: |
+        yarn install --frozen-lockfile --silent --non-interactive
+        yarn lint
 
-      - name: Slack notification FAILED
-        uses: Ilshidur/action-slack@master
-        if: failure()
-        env:
-          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
-          SLACK_AVATAR: sender # optional: repository, sender or ImageURL
-        with:
-          args: "Lint checks *FAILED* \nfrom {{ GITHUB_ACTOR }} at {{ GITHUB_REPOSITORY }}"
+    - name: Slack notification FAILED
+      uses: Ilshidur/action-slack@v2
```

Comment:
**Personal TODO :** update this once I cut a new release with these changes. I'll release it under the `2.1.0` version.